packages/react/src/components/Tile/Tile.tsx

```diff
@@ -15,17 +24,26 @@ import useIsomorphicEffect from '../../internal/useIsomorphicEffect';
 import { getInteractiveContent } from '../../internal/useNoInteractiveChildren';
 import { useMergedRefs } from '../../internal/useMergedRefs';
 
-export const Tile = React.forwardRef(function Tile(
-  { children, className, light = false, ...rest },
+export interface TileProps extends HTMLAttributes<HTMLDivElement> {
+  children?: ReactNode;
+  className?: string;
+}
+
+export const Tile = React.forwardRef<HTMLDivElement, TileProps>(function Tile(
+  {
+    children,
+    className,
+    // @ts-expect-error: Deprecated prop
+    light = false,
```

Comment:
i think this prop should still be included in the `TileProps` above

Reply:
Added & rebased :+1: